frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -48,3 +51,12 @@ export const getOCSVersion = (items: FirehoseResult): string => {
   );
   return _.get(operator, 'status.currentCSV');
 };
+
+export const getResiliencyProgress = (results: PrometheusResponse[]): number => {
+  const progress = getGaugeValue(results[0]);
```

Comment:
IMO its better to reuse existing `getInstantVectorStats`. You wont have to create your own selector

```suggestion
  const progress = getInstantVectorStats(result[0])[0];
```

Reply:
`getInstantVectorStats` is for graphs. Then we need to get the value as `getInstantVectorStats(result[0])[0].y` .This looks a bit hackish but imposes re usability. Can't we declare a common function for this in `console/shared` bc this is being used in noobaa for almost all of the cards.

Reply:
https://github.com/openshift/console/pull/2897#pullrequestreview-299829064